Tasks/DotNetCoreInstallerV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,12 +1,12 @@
 {
   "loc.friendlyName": ".NET Core SDK/runtime installer",
-  "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=853651)",
+  "loc.helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=853651)",
   "loc.description": "Acquire a specific version of the .NET Core SDK from the internet or local cache and add it to the PATH",
   "loc.instanceNameFormat": "Use .NET Core $(packageType) $(version)",
   "loc.input.label.packageType": "Package to install",
   "loc.input.help.packageType": "Please select whether to install only runtime or full SDK.",
   "loc.input.label.version": "Version",
-  "loc.input.help.version": "Specify exact version of .NET Core SDK or runtime to install.<br/>Find the value of `version-sdk` for installing SDK, or `version-runtime` for installing Runtime from any releases [here](https://github.com/dotnet/core/blob/master/release-notes/releases.json)",
+  "loc.input.help.version": "Specify exact version of .NET Core SDK or runtime to install.<br/>Find the value of `version-sdk` for installing SDK, or `version-runtime` for installing Runtime from any releases [here](https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/DotNetCoreInstallerV0/externals/releases.json)",
```

Comment:
We can also put a note here saying that for latest .Net Core versions it is recommended to use the other task.